extensions/amp-form/0.1/amp-form.js

```diff
@@ -504,8 +510,8 @@ export class AmpForm {
       this.cleanupRenderedTemplate_();
       this.setState_(FormState_.SUBMIT_ERROR);
       this.renderTemplate_(responseJson || {});
-      this.maybeHandleRedirect_(error.response);
-      user().error(TAG, `Form submission failed: ${error}`);
+      this.maybeHandleRedirect_(response);
+      user().error(TAG, `Form submission failed`);
```

Comment:
Is there any way to print the error here, or something helpful to describe how the request failed?